third_party/txt/src/minikin/FontFamily.cpp

```diff
@@ -164,6 +164,46 @@ FakedFont FontFamily::getClosestMatch(FontStyle style) const {
   return FakedFont{nullptr, FontFakery()};
 }
 
+FakedFont FontFamily::getClosestMatchWithChar(FontStyle style,
```

Comment:
The core algorithm is similar to ```FakedFont FontFamily::getClosestMatch(FontStyle style)```
I think you can make one nice method for these two.

Reply:
Ok~ Thanks!